subtests/docker_cli/login/login.py

```diff
@@ -258,8 +261,6 @@ def postprocess(self):
         cmdresult = self.sub_stuff['cmdresult']
         OutputGood(cmdresult, ignore_error=True)
         mustfail(cmdresult, 1)
-        self.failif_not_in("no successful auth challenge", cmdresult.stderr,
-                           "stderr from failed docker login")
         self.failif_not_in("401 Unauthorized", cmdresult.stderr,
                            "stderr from failed docker login")
```

Comment:
No, because: old docker = both messages, new docker = just 401. Since we don't check all output, the extra "no successful auth challenge" message from old-docker will just be ignored.